app/components/profile/following.js

```diff
@@ -100,7 +110,7 @@ const Following = ({ p2p }) => {
             title={
               following.length
                 ? 'You’ve reached the end! ✌️'
-                : 'You’re not following anybody yet 😅'
+                : <>You’re not following anybody yet. 😅 Use <StyledSearch /> Find to look up a profile</>
```

Comment:
```suggestion
                : <>You’re not following anybody yet 😅 Use <StyledSearch /> Find to look up a profile</>
```

Didn't we say in the welcome modals that emojis replace punctuation?